lib/jekyll/static_file.rb

```diff
@@ -1,6 +1,6 @@
 module Jekyll
   class StaticFile
-    attr_reader :relative_path, :extname
+    attr_reader :relative_path, :extname, :name
```

Comment:
yes, but then it breaks the tests as 'name' used to be the whole name of the file including the extension.